.ci/azure/linux.yml

```diff
@@ -431,7 +431,7 @@ jobs:
     # Skip test_onnx/test_zoo_models and test_onnx/test_backend due to long execution time
   - script: |
       export LD_LIBRARY_PATH=$(REPO_DIR)/temp/gna_03.05.00.2116/linux/x64:$(LD_LIBRARY_PATH)
-      python3 -m pytest -s $(INSTALL_TEST_DIR)/pyngraph $(PYTHON_STATIC_ARGS) \
+      python3 -m pytest -s $(INSTALL_TEST_DIR)/pyngraph $(PYTHON_STATIC_ARGS) -W ignore::FutureWarning \
```

Comment:
Do we still need to ignore it as it will show only one time? It will also ignore other FutureWarnings.

Reply:
right now there are 3 warnings, so I removed suppression